src/main/liberty/config/server.xml

```diff
@@ -0,0 +1,24 @@
+<!-- 
+ (C) Copyright IBM Corporation 2017.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<server description="Sample Servlet server">
+    <featureManager>
+        <feature>jsp-2.3</feature>
```

Comment:
Since we are doing MicroProfile lab, I would just use microprofile-2.0

Reply:
This server xml isn't actually used in the tutorial.  It exists purely for the build to prime the .m2 repo.  I will actually remove any reference to a specific feature to avoid having to change it in the future.  Thanks.